labs-ios-starter/Surveys/Model Controllers/TopicController.swift

```diff
@@ -168,6 +168,7 @@
                         completion(.failure(.badDecode))
                         return
                     }
+                    try? CoreDataManager.shared.saveContext()
```

Comment:
This is correct but I don't understand why it needs try? when it's catching everything it throws inside the method. Wonder if I could fix that

Reply:
it's a throwing method. We could keep it as throwing if the client decides they want to present CoreData errors